test/DynamoCoreWpfTests/CoreUITests.cs

```diff
@@ -797,6 +797,32 @@ public void WorkspaceContextMenu_IfSubmenuOpenOnMouseHover()
             Assert.IsTrue(currentWs.WorkspaceLacingMenu.IsSubmenuOpen);
         }
 
+        [Test]
+        [Category("UnitTests")]
+        public void TestDumpLibraryToXml()
+        {
+            ViewModel.DumpLibraryToXmlCommand.Execute(null);
+
+            // Look for library dump files aged within 10 seconds from the current time
+            var time = System.DateTime.Now;
+            var timeout = 10;
+
+            for (var t = 0; t < timeout; t++)
+            {
+                string fileName = string.Format("LibrarySnapshot_{0}.xml", time.ToString("yyyyMMddHmmss"));
+                string fullFileName = Path.Combine(Model.PathManager.LogDirectory, fileName);
+                Console.WriteLine("Looking for library dump file at " + fullFileName);
+                if (File.Exists(fullFileName))
+                {
+                    File.Delete(fullFileName);
```

Comment:
The (parameterless) command should actually write into this location, not a temporary folder:
`%APPDATA%/Dynamo/Dynamo Core/1.0/Logs/LibrarySnapshot_20160615235959.xml`

We might not be able to specify a different output folder for the test case in a straightforward manner, but I'm good to go if you feel that we don't need to delete the file after running the test case.